client/lib/application/test/access-form.test.jsx

```diff
@@ -25,25 +28,30 @@ APP_KIO = {
 
 describe('The access control form view', () => {
     var actionSpy,
+        fetchScopesStub,
         props,
         form;
 
     beforeEach(() => {
         reset();
 
+        let essentialsActions = Object.assign({}, EssentialsActions);
```

Comment:
Why do you "clone" the actions here? You can just simply stub them on the actual objects and in an afterEach() just do myStub.reset() for each of them

Reply:
yes but this is a part of current implementation and all the rest code is implemented in this manner. i would not like to completely rebuild tests at this iteration